### PR TITLE
Fix current location new2

### PR DESCRIPTION
**Pull request by chenei**

### CODE REVIEW

**Review by eilatc — APPROVED** [review]

File: public/css/map.css [hunk 1]
```diff
@@ -179,3 +174,9 @@ html, body {
     border: none;
     cursor: pointer;
 }
+
+#zoom-to-location-button > img {
+    filter: drop-shadow(0px 3px 3px rgba(0, 0, 0, 0.4));
+}
+
```
Comment (ShayMalchi) [inline public/css/map.css:181]: [minor] set your IDE to delete empty lines at end of file on file save

File: public/index.html [hunk 2]
```diff
@@ -275,6 +269,12 @@ <h1 class="pop-up-title">מקרא מפה</h1>
 
     </div>
     <!--========== map-reader-pop-up end ==========-->
+    <footer class="footer">
+            <a style="" class="nav-link map-updated">
+                <span style="font-weight: 600;">עדכון אחרון:</span>
+                <span id="last-updated-time"></span>
+            </a>
+    </footer>
```
Comment (ShayMalchi) [inline public/index.html:277]: indentation is wrong - should 4

File: public/info/index.html [hunk 3]
```diff
@@ -268,8 +268,19 @@ <h3 class="section-title">רוצים לעזור? נשמח מאוד, וגם הע
...
                         <a target="_blank" class="link" href="https://www.linkedin.com/in/zafrir-dotan/">צפריר דותן</a>,
-                        ו<a target="_blank" class="link" href="https://www.linkedin.com/in/yossi-molcho-08640498/">יוסי
-                            מולכו</a>
+                        <a target="_blank" class="link" href="https://www.linkedin.com/in/yossi-molcho-08640498/">יוסי מולכו</a>
+                    </p>
+                    <p class="small-footer-text">
+                        פיתוח:
+                        <br>
+                        <a target="_blank" class="link" href="https://www.linkedin.com/in/chen-eilat">חן אילת</a>,
+                        <a target="_blank" class="link" href="https://www.linkedin.com/in/smalchi/">שי מלאכי</a>
```
Comment (ShayMalchi) [inline public/info/index.html]: מלחי

Thank you
👑

File: public/js/app.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ const zoomToLocation = () => {
   }
 
   if (navigator.geolocation) {
-    showLoader();
+    //showLoader();
```
Comment (ShayMalchi) [inline public/js/app.js]: is this supposed to be commented out or by mistake? if you meant to comment out, i prefer you delete it along with the function if no one else calls it

**Review by ShayMalchi — APPROVED** [review]
LGTM, small changes

File: public/js/app.js (hunk 4 above)
Comment (chenei) [reply public/js/app.js]: I will delete

**Pull request merged**